### PR TITLE
Model card: T5-base fine-tuned on QuaRTz

### PR DESCRIPTION
**Pull request by mrm8488**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to the it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

### CODE REVIEW

**Comment by mrm8488** [comment]
Should I add tags like`question answering`and so on? @julien-c

**Comment by julien-c** [comment]
If it's intended to be consumed (through the inference API) as a QA model, you can add `pipeline_tag: question-answering`.

cc @Narsil

File: model_cards/mrm8488/t5-base-finetuned-quartz/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+language: en
+datasets:
+- quartz
```
Comment (julien-c) [inline model_cards/mrm8488/t5-base-finetuned-quartz/README.md:4]: pipeline_tag: question-answering

Comment (julien-c) [inline model_cards/mrm8488/t5-base-finetuned-quartz/README.md:4]:
```suggestion
- quartz
pipeline_tag: question-answering
```

**Pull request merged**